client/src/pages/reports/Form.js

```diff
@@ -440,6 +440,11 @@ const ReportForm = ({
           }
         }
 
+        tasksFilters.allUnassignedTasks = {
+          label: "All unassigned tasks",
+          queryVars: { taskedOrgUuid: null }
```

Comment:
This should at least add the filter `hasParentTask: true` like the other tasks filters.